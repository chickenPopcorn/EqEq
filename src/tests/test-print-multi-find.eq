TownCtx = {
  population = { 99; }
}

FarmCtx = {
  acres = { 42; }
}

FarmCtx:find acres {
  print("The %0.2f acres of crops is just enough\n", acres);
}

TownCtx:find population {
  print("to support the %0.0f people living in this in town\n", population);
}
