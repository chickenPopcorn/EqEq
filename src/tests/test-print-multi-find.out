The 42.00 acres of crops is just enough
to support the 99 people living in this in town
